70_ima_policy.md

```diff
@@ -0,0 +1,348 @@
+            "items": {
+                "type": "string",
+                "format": "regex"
+            }
+        },
+        "keyrings": {
+            "type": "object",
+            "patternProperties": {
+                ".*": {
+                    "type": "string",
+                    "title": "Hash of the content in the keyring"
+                }
+            }
+        },
+        "ima-buf": {
+            "type": "object",
+            "title": "Validation of ima-buf entries",
+            "patternProperties": {
+                ".*": {
+                    "type": "string",
+                    "title": "Hash of the ima-buf entry"
+                }
+            }
+        },
+        "verification-keys": {
+            "type": "array",
+            "title": "Public keys to verify IMA attached signatures",
+            "items": "string"
+        },
+        "ima": {
+            "type": "object",
+            "title": "IMA validation configuration",
+            "properties": {
+                "ignored_keyrings": {
+                    "type": "array",
+                    "title": "Ignored keyrings for key learning",
+                    "description": "The IMA validation can learn the used keyrings embedded in the kernel. Use '*' to never learn any key from the IMA keyring measurements",
+                    "items": {
+                        "type": "string",
+                        "title": "Keyring name"
+                    }
+                },
+                "log_hash_alg": {
+                    "type": "string",
+                    "title": "IMA entry running hash algorithm",
+                    "description": "The hash algorithm used for the running hash in IMA entries (second value). The kernel currently hardcodes it to sha1.",
+                    "const": "sha1"
+                }
+            },
+            "required": ["ignored_keyrings", "log_hash_alg"],
+            "additionalProperties": false
+        }
+    },
+    "required": ["meta", "release", "hashes", "excludes", "keyrings", "ima", "ima-buf", "verification-keys"],
+    "additionalProperties": false
+}
+```
+### Verifier API changes
+The option `ima_sign_verification_keys` will be removed. `allowlist` will be
+renamed to `ima_policy` and only accepts a policy formatted in the schema above.
+
+
+### Database changes
+In `verifiermain` the column `ima_sign_verification_keys` will be dropped and
+the column `allowlist` will be renamed to `ima_policy`.
+
+
+### Tenant changes
+Following changes are made to the tenant.
+
+#### New options
+* `--ima-policy`: Path to IMA policy in JSON format.
+* `--ima-policy-url`: URL to IMA policy in JSON format.
+* `--ima-policy-checksum`: SHA256 of the IMA policy.
+* `--ima-policy-sig`: Path to the signature of the IMA policy.
+* `--ima-policy-sig-url`: URL to the signature of the IMA policy.
+* `--ima-policy-sig-key`: Key to verify the signature against (in the tenant).
+
+
+#### Removal of following options
```

Comment:
Removal or deprecation?

Reply:
I added a note that we provide a better error why this options were removed.